src/diff/mount.js

```diff
@@ -29,20 +29,17 @@ export function mount(
 	startDom,
 	isHydrating
 ) {
-	let tmp,
```

Comment:
This change (and the similar one in `patch.js` are some bonus golf I discovered while working on the merge :D